tests/unit_tests/README.md

```diff
@@ -0,0 +1,36 @@
+# Run unit tests
+
+## on Linux
+
+1. Install NodeJS ([https://nodejs.org]()).
+1. Install `jq` and another necessary tools (e.g. `sed`).
+1. Open Terminal.
+1. Run `npm install`.
+1. Run `npm test`.
+
```

Comment:
For compatibility with doxygen, we might need using 1., 2., 3., ...

Reply:
Fixed in the commit [946a9a0b32ed95e838f5820b47c188f75b086223](https://github.com/polcak/jsrestrictor/pull/84/commits/946a9a0b32ed95e838f5820b47c188f75b086223).